### ISSUE TITLE
Focal active colitis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6192]
**Preferred term label:**
Focal active colitis

**Synonyms**
FAC

**Definition (free text, please give PubMed ID)**
The presence of isolated cryptitis, characterized by an inflammatory infiltrate consisting of intraepithelial neutrophils and/or neutrophils invading the lumen of the criptae, with no other microscopic alteration of the colonic mucosa and, in particular, without the presence of signs of chronic inflammation.
PMID: 29254324 

**Parent term (use hpo.jax.org/app)**
Colitis

**Comment by pnrobinson** [comment]
added as HP_0033164
